cypress/pages/login.page.ts

```diff
@@ -31,6 +31,13 @@ export const loginPage = {
     cy.get(loginLocator.password).type(signup.successful.password);
     cy.get(loginLocator.logInButton).click();
   },
+  loginBenefeciary: () => {
+    cy.fixture("login").then((signinOptions) => {
```

Comment:
The same with `login.fixture`. It's really better to create types for this fixture.